share/commands/agent-health

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+# @description monitor agent health
+# @man Check that rudder agent has no problem
+# @man +
+# @man *Options*:
+# @man +
+# @man *-n*: run in nrpe mode, print a single line and return 0,1 or 2
+# @man  put this line in your nrpe.cfg to use it
+# @man  command[check_rudder]=/opt/rudder/bin/rudder agent health -n
+
+CFE_DIR=/var/rudder/cfengine-community
+ENTERPRISE_CFE_DIR=/var/cfengine
+CFE_DISABLE_FILE=/opt/rudder/etc/disable-agent
+UUID_FILE=/opt/rudder/etc/uuid.hive
+
+[ -e "${ENTERPRISE_CFE_DIR}/bin/cf-agent" ] && CFE_DIR="${ENTERPRISE_CFE_DIR}"
+
+while getopts "n" opt; do
+  case $opt in
+    n)
+      # ignored for the moment but provided for forward compatibility
+      NRPE=y
```

Comment:
I think you should declare this variable before calling getopts and give it the empty string as a value, since you test it's not defined using -z below